server/auth/name-space/name-space.service.ts

```diff
@@ -75,7 +75,22 @@ export class NameSpaceService {
         return this.NameSpaceModel.findOne(match);
     }
 
-    getById(_id) {
+    getById(_id: string) {
+        this.validateObjectId(_id, "ID");
         return this.NameSpaceModel.findById(_id);
     }
+
+    private validateObjectId(id: string, fieldName: string): void {
```

Comment:
suggestion: this is great but should probably be on a Zod DTO pipeline. Merging to unblock production hotfix.